### DISCUSSION TITLE
EP not working

### DISCUSSION THREAD

**Discussion by ZeWeshman** [valinet/ExplorerPatcher#3616] [Q&A] [answered]
i was using the PC and i opend the star menu and it was the WIN11 one but i had it set to the WIN10 one. so i went in EP settings and i couldn't change it, the setting was not there.

so i tried coming to the github and run the installer again just in case, it killed explorer and did nothing else. i ran explorer again and got the WIN11 taskbar even tho it was set to the WIN10 one. then i tried going in the properties but they weren't there and i couldn't find them. 

i then tried to uninstall EP with BCUninstaller. it worked. 

i then ran the installer again and it killed explorer again but did nothing else.  then i came here to ask for help. here is my  windows version : Microsoft Windows Version 23H2 (build du système d'exploitaton 22631.4112) (yes my windows is in french). 

i saw another discution about the first probem but it did not help. they just said it fixed itself. 
before running the installer i tried to check for updates but it said there were none. 

now i don't even have EP anymore and i can't install it again. help

**Comment by pyrates999** [comment]
Install the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases

**Reply by ZeWeshman** [reply, marked as answer]
at first it wasn't working. but then i realized that i just neded to temporarily disable my antivirus.

**Reply by pyrates999** [reply]
see here: https://github.com/valinet/ExplorerPatcher/issues/3228

**Reply by ZeWeshman** [reply]
actually i use Avira security but i have no idea where i add exeptions to it

**Reply by pyrates999** [reply]
then you search online to try to find out
